From dd6f90f38ef9dc1f99f379e544e0f1a9379fd01e Mon Sep 17 00:00:00 2001
From: Amir Vadai <avadai@redhat.com>
Date: Mon, 3 Nov 2014 07:52:23 -0500
Subject: [infiniband] iser: Bump version to 1.4

Message-id: <5634abfcad80b36745153426ad8382c0d32c1398.1415000301.git.avadai@redhat.com>
Patchwork-id: 98694
O-Subject: [Patch RHEL7.1 25/37] IB/iser: Bump version to 1.4
Bugzilla: 1107622
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Doug Ledford <dledford@redhat.com>

From: Or Gerlitz <ogerlitz@mellanox.com>

Signed-off-by: Sagi Grimberg <sagig@mellanox.com>
Signed-off-by: Or Gerlitz <ogerlitz@mellanox.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit c7ca4b69d9ac8ec22f7c6b37a1ab9ac910c726cd)
Signed-off-by: Amir Vadai <amirv@mellanox.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/ulp/iser/iscsi_iser.h b/drivers/infiniband/ulp/iser/iscsi_iser.h
index d309620..97cd385 100644
--- a/drivers/infiniband/ulp/iser/iscsi_iser.h
+++ b/drivers/infiniband/ulp/iser/iscsi_iser.h
@@ -69,7 +69,7 @@
 
 #define DRV_NAME "iser"
 #define PFX  DRV_NAME ": "
-#define DRV_VER  "1.3"
+#define DRV_VER  "1.4"
 
 #define iser_dbg(fmt, arg...)    \
  do {      \
-- 
1.7.1